kobuki_driver/package.xml

```diff
@@ -19,18 +19,22 @@
 
   <build_depend>ecl_build</build_depend>
   <build_depend>ecl_mobile_robot</build_depend>
+  <build_depend>ecl_config</build_depend>
```

Comment:
Alpha sort. Actually refers to the line above.

Reply:
Good catch.  Fixed in 3c23381